components/raftstore/src/store/transport.rs

```diff
@@ -173,10 +174,15 @@ where
     }
 }
 
-impl<EK: KvEngine, ER: RaftEngine> LogFetchedNotifier for RaftRouter<EK, ER> {
+impl<EK: KvEngine, ER: RaftEngine> AsyncReadNotifier for RaftRouter<EK, ER> {
     #[inline]
-    fn notify(&self, region_id: u64, fetched: FetchedLogs) {
+    fn notify_fetched_logs(&self, region_id: u64, fetched: FetchedLogs) {
         // Ignore region not found as it may be removed.
         let _ = self.significant_send(region_id, SignificantMsg::RaftlogFetched(fetched));
     }
+
+    #[inline]
+    fn notify_snapshot_generated(&self, _region_id: u64, _snapshot: Box<RaftSnapshot>) {
```

Comment:
Should be in the same format as `notify_fetched_logs`.

Reply:
changed `notify_fetched_logs` to `notify_logs_fetched`.